src/test/resources/test_scripts/parameterPipeline-html-in-descriptions.groovy

```diff
@@ -0,0 +1,31 @@
+node {
+
+//
+// Commented out for now because pipeline doesn't accept Job parameters
+// on the first run of the Job, which seems a bit messed up. The actual
+// components for processing the inputs are the same in both cases, so
+// testing the inputs should be fine for now anyway.
```

Comment:
why not leave the input parameter and then add a step 04 where you press some start button for that job then you can validate the input parameter as well